ee/tabby-webserver/src/schema/setting.rs

```diff
@@ -20,6 +20,14 @@ pub struct SecuritySetting {
     pub disable_client_side_telemetry: bool,
 }
 
+impl SecuritySetting {
+    pub fn can_register_without_invitation(&self, email: &str) -> bool {
```

Comment:
```suggestion
    pub fn can_register_with_auto_invitation(&self, email: &str) -> bool {
```
Since our logic is really create an invitation for such user, maybe adjust naming into such naming